<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>word_card</class>
 <widget class="QFrame" name="word_card">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#label{
	background-color:rgb(17, 45, 78);
	border-radius: 5px;
     margin: 5px;
     border: 1px solid #dee2e6;
}
#mean_edit, 
#mean_remember_edit, 
#other_edit, 
#pronunciation_remember_edit, 
#pronunciation_edit{
	background-color:rgb(63,114,175);
	border-radius: 5px;
     margin: 5px;
     border: 1px solid #dee2e6;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignmentFlag::AlignTop">
       <widget class="QTextEdit" name="mean_edit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>请输入含义</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignTop">
       <widget class="QTextEdit" name="pronunciation_edit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>请输入读音</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignTop">
       <widget class="QTextEdit" name="mean_remember_edit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>请输入含义助记</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignTop">
       <widget class="QTextEdit" name="pronunciation_remember_edit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>请输入发音助记</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignTop">
       <widget class="QTextEdit" name="other_edit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>其他</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mean_edit</sender>
   <signal>textChanged()</signal>
   <receiver>word_card</receiver>
   <slot>mean_height_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pronunciation_edit</sender>
   <signal>textChanged()</signal>
   <receiver>word_card</receiver>
   <slot>pronunciation_height_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mean_remember_edit</sender>
   <signal>textChanged()</signal>
   <receiver>word_card</receiver>
   <slot>mean_remember_height_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pronunciation_remember_edit</sender>
   <signal>textChanged()</signal>
   <receiver>word_card</receiver>
   <slot>pronunciation_remember_height_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>other_edit</sender>
   <signal>textChanged()</signal>
   <receiver>word_card</receiver>
   <slot>other_height_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>mean_height_change()</slot>
  <slot>pronunciation_height_change()</slot>
  <slot>mean_remember_height_change()</slot>
  <slot>pronunciation_remember_height_change()</slot>
  <slot>other_height_change()</slot>
 </slots>
</ui>
